src/Assignments/Monsters/students/11th/Kangyejiii/Monsters.js

```diff
@@ -30,15 +30,38 @@ class Monsters extends Component {
 
   // SearchBox 에 props로 넘겨줄 handleChange 메소드 정의
 
+  componentDidMount() {
+    fetch("https://jsonplaceholder.typicode.com/users")
+    .then(res=>res.json())
+    .then(res=>this.setState({monsters : res}));
```

Comment:
fetch 함수 배우신대로 정말 잘 적용하셨네요! 😉

하지만, 깔끔하고 잘 작동하는 로직 외에도 정말 중요한 점이 있다면, 코드 스타일입니다.
가독성이 좋은 코드는 협업하는 팀원들로 하여금 내 코드를 읽고 이해하는데 엄청나게 많은 시간을 단축시켜주기 때문이죠 😎

그래서 해당 코드를 아래처럼 스타일링 해주실 수 있습니다!
```suggestion
    fetch("https://jsonplaceholder.typicode.com/users")
      .then(res => res.json())
      .then(res => this.setState({ monsters: res }));
```